blog/_posts/2025-01-16-advent-of-code-recap.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jamie Thompson and Seth Tisue, Scala Center
+title: Recap of Advent of Code 2024
+description: A recap of how the Scala Community participated in the recent 2024 Advent of Code challenge
+---
+
+Happy New Year everyone!
+
+**[Advent of Code](https://adventofcode.com/)**, started by [Eric Wastl](http://was.tl/), is a yearly event providing daily programming puzzles between December 1st and December 25th.
+
+Following on from [last year's blog post](https://scala-lang.org/blog/2024/01/10/advent-of-code-recap.html), let's see how the Scala community participated in the Advent of Code 2024. We were pleased by the strong engagement on both the [Scala Discord][discord] and on [our solutions website][sc-advent-of-code].
+
+## Why we do this
+
+At the [Scala Center](https://scala.epfl.ch), we love writing code in Scala, and we hope you do too. One of our core priorities is to _communicate excitement about Scala_, which motivates us to participate in the Advent of Code and share experiences solving problems with Scala with the wider programming community.
+
+Another key priority is to improve the _onboarding experience for newcomers_. Part of that experience comes from the first impressions someone has reading Scala code. We hope that through hosting articles on [our solutions website][sc-advent-of-code], newcomers can see that programming in Scala is an elegant way to solve problems. As a follow up, we aim to make this website more visible, or at least a selection from it, as a showcase for newcomers ("Scala by example").
```

Comment:
```suggestion
Another key priority is to improve the _onboarding experience for newcomers_. Part of that experience comes from the first impressions someone has reading Scala code. We hope that through hosting articles on [our solutions website][sc-advent-of-code], newcomers can see that programming in Scala is an elegant way to solve problems.
```

This is still a cool idea, but not in our roadmap anymore, as we don't have the capacity.

Unfortunately, this is not in our roadmap anymore.